main.go

```diff
@@ -442,7 +443,7 @@ func capitalize(text string) string {
 }
 
 var helperFuncs = template.FuncMap{
-	"wrap":       wrapText,
+	"wrapIndent": wrapIndent,
```

Comment:
Needed to rename, there is already a `wrap` function in the sprig FuncMap

Reply:
If this should get upstream it's also possible that you need to rename the sprig function to avoid breaking change.

Reply:
Good Point. IMO this function should never have been called `wrap` because it does more than wrapping. It wraps and indents.

The `wrap` function from sprig takes two arguments, $text and $wrapAtCharNum
The `wrap` function from calens takes three arguments, $text, $wrapAtCharNum and $indent

I see no BC compatible way out of this :-)

Reply:
But this would break existing templates.

Reply:
We could do magic Go trickery with `wrap` being a variadic function but that's really really ugly. See here: https://github.com/restic/calens/issues/2#issuecomment-567819495